Assignment2/Julen-Perez-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,58 @@
+{
+	"@context": {
+		"vi": "http://clase.com/",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"Class01": "vi:Class01",
+		"Sensor029": "Class01:Sensor029",
+		"Computer101": "Class01:Computer101",
+		"Measurement8401": "Sensor029:Measurement8401",
+		"User10A": "Computer101:User10A",
+		"includes": "vi:includes",
+		"hasTemperature": "vi:hasTemperature",
+		"hasMeasurement": "vi:hasMeasurement",
+		"time":"vi:time",
+		"hasName": "vi:hasName",
+		"hasOwner": "vi:hasOwner"
+		
+	},
+	"@id": "http://clase.com/Class01",
+	"@type": "vi:Class01",
+	"includes": [
+		{
+			"Sensor029":{
```

Comment:
"Sensor029": isn't needed. I suggest you remove it. Also, delete "Measurement8401":, "Computer101":, and "User10A":
Be careful with the curly braces.